### PR TITLE
Add UI toggles

### PR DESCRIPTION
**Pull request by mohsinmalik324**
Adding in toggles to UI elements so that we can reliably hide/modify certain UI elements without them getting wiped when merging from upstream

### CODE REVIEW

**Comment by mohsinmalik324** [comment]
I have read the CLA Document and I hereby sign the CLA

File: frontend/src/lib/components/home/ItemsList.svelte [hunk 1]
```diff
@@ -68,7 +70,7 @@
 				...script
 			}
 		})
-
+		console.log('set loading=false')
```
Comment (rubenfiszel) [inline frontend/src/lib/components/home/ItemsList.svelte]: let's remove the console logs

File: frontend/src/lib/components/home/ItemsList.svelte [hunk 2]
```diff
@@ -87,6 +88,7 @@
 				...x
 			}
 		})
+		console.log('set loading=false')
```
Comment (rubenfiszel) [inline frontend/src/lib/components/home/ItemsList.svelte]: rm

File: frontend/src/lib/components/home/ItemsList.svelte [hunk 3]
```diff
@@ -100,7 +102,7 @@
 				...app
 			}
 		})
-
+		console.log('set loading=false')
```
Comment (rubenfiszel) [inline frontend/src/lib/components/home/ItemsList.svelte]: rm

File: frontend/src/lib/components/home/ItemsList.svelte [hunk 4]
```diff
@@ -216,6 +218,8 @@
 	$: items && resetScroll()
 
 	let archived = false
+
+	console.log(`filteredItems=${filteredItems.length}`)
```
Comment (rubenfiszel) [inline frontend/src/lib/components/home/ItemsList.svelte]: rm

**Review by rubenfiszel — DISMISSED** [review]

**Pull request merged**